canonical-websites/maas.io
# Update index page styling and copy

## Done

- Updated copy according to [copydoc](https://docs.google.com/document/d/1Gv3_A3vW2k7U-0B0KVSHkEnGwgtIlDl5P1jGMSc8sqI/edit#)
- Updated styles (taken from #274)

## QA

- Pull code, run `./run`, navigate to http://0.0.0.0:8006/
- Check that the copy matches the [copydoc](https://docs.google.com/document/d/1Gv3_A3vW2k7U-0B0KVSHkEnGwgtIlDl5P1jGMSc8sqI/edit#)
- Note that the buttons on the hero strip (Install MAAS, See how it works) don't match the copydoc because there isn't currently a `/compare` page yet. Also, the logos in the Key Features matrix are wrong and will be updated when the icons have been finalised. **EDIT:** new icons have been added now
- Check page on large and small screens

## Issue / Card

Fixes #268, fixes #286
